Demo/Fullscreen/LoginEntry/LoginEntryViewDemoView.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import FinniversKit
+
+public class LoginEntryViewDemoView: UIView {
+    private lazy var loginEntryView: LoginEntryView = {
+        let loginEntryView = LoginEntryView()
+        loginEntryView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
You could use the FinniversKit initializer instead for `UIView`'s. That saves you the annoying line `x.translatesAutoresizingMaskIntoConstraints = false` 🎉

I think it applies to every view we have in FK, except for `Label` and `Button` which has their own init.

I.e.:
```swift
let loginEntryView = LoginEntryView(withAutoLayout: true)
```

Reply:
👍